pkg/custompluginmonitor/plugin/plugin.go

```diff
@@ -234,7 +238,11 @@ func (p *Plugin) run(rule cpmtypes.CustomRule) (exitStatus cpmtypes.Status, outp
 	output = string(stdout)
 	output = strings.TrimSpace(output)
 
-	if cmd.ProcessState.Sys().(syscall.WaitStatus).Signaled() {
+	m.Lock()
+	cmdKilled := timeout
```

Comment:
Actually it is possible that the command exit itself after we mark time, and the kill didn't actually work. That is why I prefer `timeout` than `killed`.

Let's just keep using `timeout`.

Reply:
This is not as important, we can always change it later if needed.